src/server/build-server.ts

```diff
@@ -64,7 +64,10 @@ export function buildServer(): FastifyInstance {
     }
 
     const token = jwt.sign({ id: user.id }, process.env.TOKEN_KEY, {
-      expiresIn: Number(process.env.TOKEN_TIMEOUT) || 30,
+      expiresIn:
+        rememberMe && rememberMe == true
+          ? "1w"
+          : Number(process.env.TOKEN_TIMEOUT) || 30,
```

Comment:
```suggestion
        rememberMe ? "1w" : +(process.env.TOKEN_TIMEOUT ?? 30),
```

A chagem com apenas `rememberMe` no Javascript só passa se ele for true mesmo. E o `+` é uma alternativa ao `Number()`